module-1/views/contacts-admin.ejs

```diff
@@ -36,7 +34,7 @@
 		<%})%>
 	</div> -->
 	
-	<div class="table">
+	<div id="aaa" class="table">
```

Comment:
`aaa` is not an appropriate class name, use something meaningful

Reply:
Deleted